src/Symfony/Bridge/Doctrine/Test/DoctrineTestHelper.php

```diff
@@ -14,6 +14,7 @@
 use Doctrine\Common\Annotations\AnnotationReader;
 use Doctrine\ORM\Mapping\Driver\AnnotationDriver;
 use Doctrine\ORM\EntityManager;
+use PHPUnit\Framework\TestCase;
```

Comment:
Actually, with the constraint  `<4.8.35|<5.4.3,>=5.0`, the real QCN is [`phpunit\framework\TestCase`](https://github.com/sebastianbergmann/phpunit/blob/5.4.2/src/ForwardCompatibility/TestCase.php#L11).